### ISSUE TITLE
Wrong requirement for Joplin Android listed in Google Play Store

### ISSUE THREAD

**Issue by philip-n** [laurent22/joplin#4585]
tl;dr: Please update the requirement for Joplin in the Play Store, as 4.x is not supported any more.

---

Google play store currently lists "Android 4.1 or higher" as requirement for installing Joplin. However, Android 4.4 is not supported anymore due to an underlying error in React Native. This leads to situations where users with an older version of Joplin on their Android 4 decide to update, and are left with a non-working app (see https://github.com/laurent22/joplin/issues/4507 and unreported cases, e.g. me).

The technical details have been analyzed and discussed in https://github.com/laurent22/joplin/issues/4057

Result was  @laurent22 s decision:
> That's unfortunate but it means we don't support Android 4 anymore then.

## Environment

Joplin version: current
Platform: Android
OS specifics: 4.4.2

## Steps to reproduce

1. Use an old version of Joplin on an old Android 4.x phone.
2. Use Google Play Store to update the app.
3. The new version has been installed, but is now broken :/

## Describe what you expected to happen
Google Play Store has the correct version requirement and does not allow to update the app on 4.x Android devices.

## Logfile
See https://github.com/laurent22/joplin/issues/4057 for the underlying technical issue and logfiles.

**Comment by tessus** [comment]
I don't have access to the PlayStore meta info for Joplin, but you are right. The min req should be set to Android 5.
